src/ViewInstance.js

```diff
@@ -523,236 +493,80 @@ class ViewInstance extends React.Component {
-      instanceTypeId: get(instance, ['instanceTypeId'], ''),
-      natureOfContentTerms: get(instance, 'natureOfContentTermIds', []),
-      instanceFormatIds: get(instance, ['instanceFormatIds'], []),
-      languages: get(instance, ['languages'], []),
-      publicationFrequency: get(instance, ['publicationFrequency'], []),
-      publicationRange: get(instance, ['publicationRange'], []),
-    };
-
-    const electronicAccess = get(instance, ['electronicAccess'], []);
-
-    const subjects = get(instance, ['subjects'], []);
-
-    const instanceRelationship = {
-      childInstances: get(instance, ['childInstances'], []),
-      parentInstances: get(instance, ['parentInstances'], []),
-    };
-
-    const initialAccordionsState = {
-      acc01: !areAllFieldsEmpty(values(instanceData)),
-      acc02: !areAllFieldsEmpty(values(titleData)),
-      acc03: !areAllFieldsEmpty([identifiers]),
-      acc04: !areAllFieldsEmpty([contributors]),
-      acc05: !areAllFieldsEmpty(values(descriptiveData)),
-      acc06: !areAllFieldsEmpty([instance?.notes ?? []]),
-      acc07: !areAllFieldsEmpty([electronicAccess]),
-      acc08: !areAllFieldsEmpty([subjects]),
-      acc09: !areAllFieldsEmpty([get(instance, 'classifications', [])]),
-      acc10: !areAllFieldsEmpty(values(instanceRelationship)),
-    };
-
     return (
-      <Pane
-        data-test-instance-details
-        style={{ flex: 'auto' }}
-        appIcon={<AppIcon app="inventory" iconKey="instance" />}
-        paneTitle={
-          <span data-test-header-title>
-            <FormattedMessage
-              id="ui-inventory.instanceRecordTitle"
-              values={{
-                title: instance.title,
-                publisherAndDate: this.getPublisherAndDate(instance),
-              }}
-            />
-          </span>
-        }
-        paneSub={instanceSub()}
-        dismissible
-        onClose={onClose}
-        actionMenu={this.createActionMenuGetter(instance)}
-      >
-        <TitleManager record={instance.title} />
-        <AccordionStatus>
-          <Row end="xs">
-            <Col
-              data-test-expand-all
-              xs
-            >
-              <ExpandAllButton />
-            </Col>
-          </Row>
-
-          <InstanceTitle
-            instance={instance}
-            instanceTypes={referenceTables.instanceTypes}
-          />
-
-          <AccordionSet initialStatus={initialAccordionsState}>
-            {
-              (!holdingsrecordid && !itemid) ?
-                (
-                  <Switch>
-                    <Route
-                      path="/inventory/viewsource/"
-                      render={() => (
-                        <this.cViewMarc
+      <>
+        <InstanceDetails
+          onClose={onClose}
+          actionMenu={this.createActionMenuGetter(instance)}
+          instance={instance}
+          referenceData={referenceTables}
+        >
+          {
+            (!holdingsrecordid && !itemid) ?
```

Comment:
```suggestion
            (!holdingsrecordid && !itemid) &&
```

Reply:
will skip for now as these blocks will be moved from `ViewInstance` in next PRs